src/mmw/apps/modeling/tasks.py

```diff
@@ -64,7 +66,7 @@ def start_histogram_job(self, json_polygon):
 
     return {
         'pixel_width': aoi_resolution(polygon),
-        'sjs_job_id': histogram_start([json_polygon], self.retry)
+        'sjs_job_id': histogram_start([json_polygon], self.retry),
```

Comment:
Sorry to be so persistent here, but we don't need this extra comma here.

Essentially what I'm trying to do is look at all the lines changed in this PR and ask if this line _must_ change for the purpose of this PR. If not, even though it might be a good idea (like having trailing commas in objects, which is objectively a good idea), I would prefer to not include it. This allows PRs to be atomic and isolated, because they have all the changes they need and none of the changes they don't.